hack/build-index-image.sh

```diff
@@ -35,22 +34,64 @@ function create_index_image() {
   BUNDLE_IMAGE_NAME="${IMAGE_REGISTRY}/${REGISTRY_NAMESPACE}/${BUNDLE_REGISTRY_IMAGE_NAME}:${CURRENT_VERSION}"
   INDEX_IMAGE_NAME="${IMAGE_REGISTRY}/${REGISTRY_NAMESPACE}/${INDEX_REGISTRY_IMAGE_NAME}:${CURRENT_VERSION}"
 
-  INDEX_IMAGE_PARAM=
-  if [[ -n ${2} ]]; then
-    PREV_INDEX_IMAGE="${IMAGE_REGISTRY}/${REGISTRY_NAMESPACE}/${INDEX_REGISTRY_IMAGE_NAME}:${PREV_VERSION}"
-    INDEX_IMAGE_PARAM=--from-index="${PREV_INDEX_IMAGE}"
-  fi
-
   podman build -t "${BUNDLE_IMAGE_NAME}" -f bundle.Dockerfile --build-arg "VERSION=${CURRENT_VERSION}" .
   podman push "${BUNDLE_IMAGE_NAME}"
 
   # Referencing the unstable bundle digest in the index image, rather than the floating tag, to avoid
   # unalignment between cached index image and fetched bundle image.
   BUNDLE_IMAGE_NAME=$("${PROJECT_ROOT}/tools/digester/digester" --image "${BUNDLE_IMAGE_NAME}")
 
-  # shellcheck disable=SC2086
-  ${OPM} index add --bundles "${BUNDLE_IMAGE_NAME}" ${INDEX_IMAGE_PARAM} --tag "${INDEX_IMAGE_NAME}" -u podman --mode semver
+
+  # File-Based Catalog handling
+  ${OPM} migrate ${INDEX_IMAGE_NAME} fbc-catalog || true
```

Comment:
I think we should avoid writing to the root directory of the project. What do you think about output to `_out/fbc-catalog`?

If you do that, please add an env var for this path, in order to use it later in the script.

Reply:
I think it would be better to change the working directory in this part to `${PROJECT_ROOT}/_out` to avoid adding the `_out/ ` prefix for every file reference.